Fixed link format + used GPT Editor (#1207)
wandb/docs: content/tutorials/integration-tutorials/tensorflow_sweeps.md

Use W&B for machine learning experiment tracking, dataset versioning, and project collaboration.

{{< img src="/images/tutorials/huggingface-why.png" alt="" >}}

Use W&B Sweeps to automate hyperparameter optimization and explore model possibilities with interactive dashboards:

{{< img src="/images/tutorials/tensorflow/sweeps.png" alt="" >}}

## Why use sweeps

* **Quick setup**: Run W&B sweeps with a few lines of code.
* **Transparent**: The project cites all algorithms used, and the [code is open source](https://github.com/wandb/wandb/blob/main/wandb/apis/public/sweeps.py).
* **Powerful**: Sweeps provide customization options and can run on multiple machines or a laptop with ease.

For more information, see the [Sweep documentation]({{< relref "/guides/models/sweeps/" >}}).

## What this notebook covers

* Steps to start with W&B Sweep and a custom training loop in TensorFlow.
* Finding best hyperparameters for image classification tasks.

**Note**: Sections starting with _Step_ show necessary code to perform a hyperparameter sweep. The rest sets up a simple example.

## Install, import, and log in

### Install W&B

```bash

> %%capture

Reviewer: ```suggestion
```

> !pip install wandb

Reviewer: ```suggestion
pip install wandb
```

import matplotlib.pyplot as plt
```

```python
import wandb
from wandb.integration.keras import WandbMetricsLogger

wandb.login()
```

> > Note: If you are new to W&B or not logged in, the link after running `wandb.login()` directs to the sign-up/login page. Sign-up involves a few clicks.

Reviewer: ```suggestion
{{< alert >}}
If you are new to W&B or not logged in, the link after running `wandb.login()` directs to the sign-up/login page.
{{< /alert >}}
```
